alerting/src/main/kotlin/org/opensearch/alerting/action/GetSuggestionsAction.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.action
+
+import org.opensearch.action.ActionType
+
+class GetSuggestionsAction private constructor() : ActionType<GetSuggestionsResponse>(NAME, ::GetSuggestionsResponse) {
+    companion object {
+        val INSTANCE = GetSuggestionsAction()
+        const val NAME = "cluster:admin/opensearch/alerting/suggestions/get" // TODO: is this an ok name to use?
```

Comment:
This was one of the first pieces of code that I wrote when I was still learning about REST handler structures. I essentially just copied the format of another `NAME` and put it here as a placeholder, but I've been meaning to ask you guys about what this value does. To remove its association with alerting, should I just delete the `/alerting` part of the string? Should I also change the `get` at the end to a `post` since that's the verb we are supporting now?